WireDomain/Sources/WireDomain/Event Processing/UserEventProcessor/UserPushRemoveEventProcessor.swift

```diff
@@ -30,9 +31,10 @@ protocol UserPushRemoveEventProcessorProtocol {
 
 struct UserPushRemoveEventProcessor: UserPushRemoveEventProcessorProtocol {
 
+    let repository: any UserRepositoryProtocol
+
     func processEvent() async throws {
```

Comment:
does that makes sense to clean the signature then:
```suggestion
    func processEvent() {
```

Reply:
fixed thx